### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by serviceace**

### CODE REVIEW

File: src/Components/App/styles.ts [hunk 1]
```diff
@@ -10,22 +10,22 @@ const styles = StyleSheet.create({
     justifyContent: "center",
     alignItems: "center",
   },
-  flex_container: {
+  flexContainer: {
     display: "flex",
     flexDirection: "column",
   },
-  main_container: {
+  mainContainer: {
```
Comment (jafidiesel) [inline src/Components/App/styles.ts:17]: remove this empty class

File: src/Components/BtnSearch/styles.ts [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import { StyleSheet } from "aphrodite";
 
 const styles = StyleSheet.create({
+<<<<<<< HEAD
+    btnSearch: {
+=======
     btn_search: {
+>>>>>>> 4b083e8a6ed1561c3ce59dc491a7068bae30343d
```
Comment (jafidiesel) [inline src/Components/BtnSearch/styles.ts:8]: review this

File: src/Components/Card/styles.ts [hunk 3]
```diff
@@ -30,14 +30,14 @@ const styles = StyleSheet.create({
 	        transform: 'translateY(-3%)',
           },
       },
-    card__contenido : {
+    cardContent : {
```
Comment (jafidiesel) [inline src/Components/Card/styles.ts:33]: remove this

File: src/Components/Card/styles.ts [hunk 4]
```diff
@@ -30,14 +30,14 @@ const styles = StyleSheet.create({
...
 
     },
-    nombre: {
+    spotName: {
         backgroundColor: '#c2eaba',
         padding: '10px',
     },
-    materials: {
+    CardMaterials: {
```
Comment (jafidiesel) [inline src/Components/Card/styles.ts:40]: cardMaterials

File: src/Components/Table/styles.ts [hunk 5]
```diff
@@ -2,7 +2,10 @@ import { StyleSheet } from "aphrodite";
 
 const styles = StyleSheet.create({
     table: {
+<<<<<<< HEAD
+=======
```
Comment (jafidiesel) [inline src/Components/Table/styles.ts:6]: review this

File: src/Components/TopBar/styles.ts [hunk 6]
```diff
@@ -3,14 +3,17 @@ import { StyleSheet } from "aphrodite";
...
+    width: '100%',
     padding: "7px",
     boxSizing: "border-box",
     display: "flex",
     justifyContent: "space-between",
     alignItems: "center",
     boxShadow: "0px 10px 10px -8px gray",
     marginBottom: 20,
+    position: 'fixed',
+    zIndex: 5,
```
Comment (jafidiesel) [inline src/Components/TopBar/styles.ts:16]: is this zIndex necesary? otherwise remove it

File: tp [hunk 7]
```diff
@@ -0,0 +1,190 @@
+[33mcommit 3b041ce7674bdb7e12c0e01254aa5f761eee50fe[m[33m ([m[1;36mHEAD[m[33m -> [m[1;32mtest-branch[m[33m, [m[1;31morigin/test-branch[m[33m)[m
```
Comment (jafidiesel) [inline tp:1]: delete this file

**Review by jafidiesel — CHANGES_REQUESTED** [review]

File: src/Components/TopBar/styles.ts (hunk 6 above)
Comment (serviceace) [reply src/Components/TopBar/styles.ts:16]: It is necessary for the TopBar to function

**Review by jafidiesel — APPROVED** [review]

**Pull request merged**